### ISSUE TITLE
Using renv without R projects

### ISSUE THREAD

**Issue by hreypar** [rstudio/renv#460]
Hello, 
so, my question is: is it possible to use R **without** creating an R project? or is it mandatory? 

thanks!

**Comment by kevinushey** [comment]
While `renv` is built to encourage a project-centric workflow, it's flexible enough that it can be adapted to other workflows as well.

What exactly do you want to do?

**Comment by hreypar** [comment]
> While `renv` is built to encourage a project-centric workflow, it's flexible enough that it can be adapted to other workflows as well.
> 
> What exactly do you want to do?
Hey, thanks for replying! 

What I want to do, exactly, is use something that let's me keep an R library ([multiHiCCompare](https://github.com/dozmorovlab/multiHiCcompare)) within [this module](https://github.com/hreypar/filter-hicexp-results
) that I programmed so that I can clone it into a different computer (e.g. my institute's cluster) and it works. 

I tried using `packrat`, but it couldn't find any sources for some of the dependency libraries for `multiHiCCompare`. I tried solutions that I found online or around github (like using `devtools` within `packrat` as an external library), but it wouldn't work either. 

So then I finally discovered `renv` and thought it was perfect because it can use Bioconductor and github to install libraries (if one attempts to install `multiHiCCompare` using `install.packages("multiHiCCompare")` it is a nightmare (due to the dependencies), but doing it from its github repo works fine, but then, when I initialize renv, it creates files for an Rstudio R project and I definitely do not want that. 

My code is simple and command line oriented; the only time I use `R` interactively is when I am programming my scripts, so I'd really like to find out a solution to port `multiHiCCompare`.

cheers!
